### ISSUE TITLE
panctl : ModuleNotFoundError: No module named 'prompt_toolkit.eventloop.defaults'

### ISSUE THREAD

**Issue by arthurlutz** [matrix-org/pantalaimon#33]
Following #32 I removed the globalpackages from my virtualenv and re-did a pip install. 

Ended up getting : 

```
 panctl 
Traceback (most recent call last):
  File "/home/arthur/.virtualenvs/pantalaimon/bin/panctl", line 5, in <module>
    from pantalaimon.panctl import main
  File "/home/arthur/.virtualenvs/pantalaimon/lib/python3.6/site-packages/pantalaimon/panctl.py", line 30, in <module>
    from prompt_toolkit.eventloop.defaults import use_asyncio_event_loop
ModuleNotFoundError: No module named 'prompt_toolkit.eventloop.defaults'

```

**Comment by arthurlutz** [comment]
```
pip list | grep prompt
prompt-toolkit     3.0.2  

```

**Comment by arthurlutz** [comment]
`pip install "prompt-toolkit<3.0.0"` solved it.

**Comment by poljar** [comment]
This has been fixed by #31.
